raiden/tests/unit/conftest.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+# from raiden.tests.integration.fixtures.blockchain import *  # noqa: F401,F403
```

Comment:
What's this file for? It just seems to have 1 commented import line and nothing else

Reply:
that's slipped in when rebasing the proxy unit tests. It can go of course.